.scripts/clean-autopr.ts

```diff
@@ -5,46 +5,53 @@
  */
 
 import { exec as execWithCallback } from "child_process";
-import { getAuthenticatedClient } from "./github";
-import { PullRequestsGetAllParams } from "@octokit/rest";
+import Octokit from "@octokit/rest";
 
-const _repositoryOwner = "Azure";
+export function getToken(): string {
+    const token: string = process.env.SDK_GEN_GITHUB_TOKEN || "";
+    _validatePersonalAccessToken(token);
+
+    return token;
+}
+
+function _validatePersonalAccessToken(token: string): void {
+    if (!token) {
+        const text =
+            `Github personal access token was not found as a script parameter or as an
+        environmental variable. Please visit https://github.com/settings/tokens,
+        generate new token with "repo" scope and pass it with -token switch or set
```

Comment:
You mention that a user can pass the token with the `-token` switch, but in the `getToken()` function it looks like it can only be passed in via an environment variable. Am I missing something?